src/js/browser-plugin.js

```diff
@@ -314,14 +307,15 @@ const publishTweetAlertMisinfo = (label, url, tweetText) => {
   let popupTitle = browserAPI.i18n.getMessage('content_tagged_as', auxlabel);
   let popupButtonText = browserAPI.i18n.getMessage('ok');
   
+  log2Server('publish tweet', url, `Tweet id: ${tweetText}`, 'detected "not credible" tweet before publish'); 
   return Swal2.fire({
     type: 'info',
     title: popupTitle,
     showCloseButton: true,
     showCancelButton: false,
     confirmButtonColor: buttonColor,
     confirmButtonText: popupButtonText,
-    onClose: sendCloseAwaitPopUp(url, tweetText),
+    onClose: sendCloseAwaitPopUp,
```

Comment:
If we do like this, then when the "sendCloseAwaitPopUp" function will be executed, the parameters will be empty. I think it is better to just copy the function code here: like, for example, how it is done in the "preConform" action in the popup opened on line 1353.